lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -382,52 +374,55 @@
                     return deferred.promise();
                 },
 
-                createViewWithHeader: function (options) {
-                    var router = this.router,
-                        breadcrumbs, headerView,
-                        viewDescription, viewTitle;
-                    breadcrumbs = [{
+                createBreadcrumbs: function(topic, team) {
+                    var breadcrumbs = [{
                         title: gettext('All Topics'),
                         url: '#browse'
                     }];
-                    if (options.parentTopic) {
+                    if (topic) {
                         breadcrumbs.push({
-                            title: options.parentTopic.get('name'),
-                            url: '#topics/' + options.parentTopic.id
+                            title: topic.get('name'),
+                            url: '#topics/' + topic.id
                         });
                     }
-                    if (options.parentTeam) {
+                    if (topic && team) {
```

Comment:
Nit: push this down into the `if (topic)` block, since there we already know that the first argument to the `&&` is truthy.